src/main/java/com/autotune/Autotune.java

```diff
@@ -59,6 +61,9 @@ public class Autotune {
 
     public static void main(String[] args) {
 
+        // Turning off the logging level for the specific package to reduce console logging
```

Comment:
Include a try and catch block. If we transition to a different package, this action may result in an error.